README.md

```diff
@@ -69,6 +69,33 @@ yarn playground
 - [`/src/index.ts`](https://github.com/graphcool/graphql-boilerplate/blob/master/src/index.ts) is the entry point of your application, pulling everything together and starting the [`graphql-yoga`](https://github.com/graphcool/graphql-yoga) Server.
 - [`/src/resolvers/`](https://github.com/graphcool/graphql-boilerplate/tree/master/src/resolvers) includes the actual business logic of your application. In GraphQL you implement [resolvers](http://graphql.org/learn/execution/) that *resolve*  a specific query being requested
 
+### Next Steps
+#### Adding a custom resolver
+Suppose we want to add a custom resolver to delete a post (example). We can take the following steps:-
+
+Add `delete` field to the Mutation type in `src/schema.graphql`
+```
+type Mutation {
+  createDraft(title: String!, text: String): Post
+  publish(id: ID!): Post
+  delete(id: ID!): Post
+}
+```
+
+Add `delete` resolver to Mutation part of `src/index.js`
+```
+delete(parent, { id }, ctx, info) {
+  return ctx.db.mutation.deletePost(
+  {
+    where: { id }
+  },
+    info
+  );
+}
+```
+
+Run `yarn start`
+
```

Comment:
Add the example how to add a post:

```graphql
mutation {
  delete(id: "cuid") {
    id
  }
}
```

Reply:
I suppose you mean, how to delete a post? Made the required changes. Let me know if anything else needed :)